src/components/MentionSuggestions.tsx

```diff
@@ -85,9 +76,9 @@ function MentionSuggestions(props) {
                     style={[styles.mentionSuggestionsText, styles.flexShrink1]}
                     numberOfLines={1}
                 >
-                    {_.map(styledDisplayName, ({text, isColored}, i) => (
+                    {styledDisplayName.map(({text, isColored}) => (
                         <Text
-                            key={`${text}${i}`}
+                            key={`${text}${isColored}`}
```

Comment:
Are you sure this is a proper key? Looks like adding index avoids duplicates

Reply:
reverted